app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/CheckDuplicateDeclarationDetailsController.scala

```diff
@@ -48,9 +53,10 @@ class CheckDuplicateDeclarationDetailsController @Inject() (
   implicit val duplicateDeclarationExtractor: DraftClaim => Option[DisplayDeclaration] =
     _.duplicateDisplayDeclaration
 
+  val isDuplicate: Boolean = true
```

Comment:
Somehow this isDuplicate stuff seems not right

Reply:
@aneksamun This controller is specific for handling the case when user selects duplicate MRN option from the previous page.  The val was just extracted from a method so it can be reused.